core/main_dispatcher_bot.py

```
@@ -0,0 +1,213 @@
+                            else:
+                                await self.app.client.chat_postMessage(
+                                    channel=channel, 
+                                    text=f"❌ Audio transcription failed.", 
+                                    thread_ts=ts
+                                )
+                        except Exception as e:
+                            logger.error(f"Audio processing error: {e}")
+                    
+                    # Handle video files
+                    elif mimetype.startswith('video/'):
+                        user = event.get('user')
```

Ruff (0.12.2)

Local variable `user` is assigned to but never used

Remove assignment to unused variable `user`

(F841)